### PR TITLE
github-pages monitor

### PR DESCRIPTION
**Pull request by ivy-rew**

### CODE REVIEW

**Review by alexsuter — APPROVED** [review]
Already cool stuuf? :)

Let's goo! You may ignore this repo too!

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Generate Monitor
+      run: |
+        pwd
+        ls -la
```
Comment (jackra1n) [inline .github/workflows/ci.yml]: `pwd` and `ls -la` needed? 🤔

Comment (alexsuter) [reply .github/workflows/ci.yml]: seems like he was on an investigation trip to find the build.sh!

Comment (ivy-rew) [reply .github/workflows/ci.yml]: yes, exactly .. missed the prefix `./`and this is how I got there :)

**Comment by ivy-rew** [comment]
> Already cool stuuf? :)
> 
> Let's goo! You may ignore this repo too!

partly .... we will have more contributors on the market in the next 2 days. so it's worth spreading that stuff

**Pull request merged**